### PR TITLE
#165412881 like and deslike article

### PR DESCRIPTION
**Pull request by sengayire**
#### What does this PR do?
- allow users to Like and Dislike an article
#### Description of Task to be completed?
- user can like  or dislike an article and also user can be able to remove his/her reaction
- fix search article by tag issue
#### How should this be manually tested?
- visit: https://ninjas-ah-staging.herokuapp.com 
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- #165412881
#### Screenshots (if appropriate)
- N/A
#### Questions:
-N/A

### CODE REVIEW

File: src/components/Articles/LikeArticle/LikeArticle.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp, faThumbsDown } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+import { getArticleLikes, likeArticle, dislikeArticle } from '../../../actions';
+import { Button } from '../../common';
+import { clearHighlightArticleStore } from '../../../actions/articles';
+import './LikeArticle.scss';
+
+const style = { color: '' };
```
Comment (luc-tuyishime) [inline src/components/Articles/LikeArticle/LikeArticle.js]: Why do you have a style object here?

Comment (kagaramag) [reply src/components/Articles/LikeArticle/LikeArticle.js]: Remove this style object if you are not using it

**Review by kagaramag — APPROVED** [review]

**Review by luc-tuyishime — APPROVED** [review]

File: src/components/Chat/Chat.scss [hunk 2]
```diff
@@ -56,8 +56,9 @@ $font-size-floating-chat: 25px;
     cursor: pointer;
   }
   .modal-dialog {
+    position: relative;
     overflow-y: initial !important;
-    z-index: 200;
+    z-index: 100000000;
```
Comment (e-liyai) [inline src/components/Chat/Chat.scss]: this shows an unclear understanding of the css layout

File: src/components/Articles/LikeArticle/LikeArticle.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+  render() {
+    const { likesNumber, dislikesNumber, likeClicked, dislikeClicked } = this.state;
+    const {
+      isAuth,
+      article: { slug }
+    } = this.props;
+    return (
+      <div className="row">
+        <Button
```
Comment (e-liyai) [inline src/components/Articles/LikeArticle/LikeArticle.js]: could we abstract this since the two buttons are almost similar

Comment (sengayire) [reply src/components/Articles/LikeArticle/LikeArticle.js]: I tried to abstract the buttons

Comment (e-liyai) [reply src/components/Articles/LikeArticle/LikeArticle.js]: how have you abstracted this?

**Review by e-liyai — APPROVED** [review]

**Pull request merged**